presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -2,6 +2,38 @@
 Deploying Presto
 ================
 
+.. _requirements:
```

Comment:
I used it for the Java requirement and though we should have the link for overall as well since we will probably link to it soon. I just did not actively look where to use that link yet. I can remove and defer to later if you want.

Reply:
Let's remove them unless they are actually used.

Reply:
Done